internal/routes.go

```diff
@@ -130,7 +130,11 @@ func processMedia(c *gin.Context) {
 		c.JSON(http.StatusInternalServerError, gin.H{"info": fmt.Sprint(task.TaskUID) + ": " + err.Error()})
 	}
 
-	c.JSON(http.StatusOK, gin.H{"info": fmt.Sprintf("Fetched and inserted %d media into meilisearch and skipped %d", len(medias), failedMedia)})
+	c.JSON(http.StatusOK, gin.H{
+		"info":       fmt.Sprintf("Fetched and inserted %d media into meilisearch and skipped %d", len(medias), failedMedia),
+		"successful": len(medias),
+		"skipped":    failedMedia,
```

Comment:
Don't they need to be printed somewhere in Python?

Reply:
They are - check the pbar.set_postfix()

Reply:
Yeah they are printed in the progress bar, but not from the json sent from Go -> Python

Reply:
not sure if we need them, if we can see it the numbers in the `info` field in the json?

Reply:
wait i'm stupid as hell, my bad xD